server/helpers/adapters_tracker.go

```diff
@@ -75,6 +78,7 @@ func (a *AdaptersTracker) DeployAdapter(ctx context.Context, adapter models.Adap
 	}
 
 	// Deploy to current platform
+	targetIP := "127.0.0.1"
```

Comment:
Have you confirmed that this change isn't going to cause a regression issue with Docker as the platform? What's the need for change from "localhost"?